generate-renode-scripts.py

```diff
@@ -362,28 +336,61 @@ def generate_repl():
         }
     }
 
-    shadow_base = (configuration.constants['shadow_base']['value']
-                   if 'shadow_base' in configuration.constants
-                   else None)
-
-    for mem_region in configuration.mem_regions.values():
-        if mem_region['name'] not in non_generated_mem_regions:
-            result += generate_memory_region(mem_region, shadow_base)
+    # RISC-V CPU in Renode requires memory region size
+    # to be a multiple of 4KB
+    for mem_region in filter_memory_regions(list(configuration.mem_regions.values()), alignment=0x1000):
+        result += generate_memory_region(mem_region)
 
     result += generate_cpu('cpu_timer' if 'cpu' in configuration.peripherals else None)
 
     for name, peripheral in configuration.peripherals.items():
         if name not in name_to_handler:
-            print('Skipping unsupported peripheral {} at {}'
-                  .format(name, peripheral['address']))
+            print('Skipping unsupported peripheral `{}` at {}'
+                  .format(name, hex(peripheral['address'])))
             continue
 
         h = name_to_handler[name]
-        result += h['handler'](peripheral, shadow_base, **h)
+        result += h['handler'](peripheral, **h)
 
     return result
 
 
+def filter_memory_regions(raw_regions, alignment=None):
+    """ Filters memory regions skipping those
```

Comment:
The wrapping on this docstring is really weird.

Reply:
Fixed.